### DISCUSSION TITLE
Please add 2fa (TOTP) for better security

### DISCUSSION THREAD

**Discussion by drgimpfen** [jonaswinkler/paperless-ng#972] [Feature Requests]
As i try to store most of my documents and they contain alot of sensitiv information i would like to have 2-factor-auth with TOTP so i can use Authy on my phone.

**Comment by jonaswinkler** [comment]
Search the repo for Authelia and remote user authentication. Someone already did that using additional services.

**Reply by jonaswinkler** [reply]
Also, #142
